test/arithmetic/divider_test.dart

```diff
@@ -479,6 +568,7 @@ class MultiCycleDividerTest extends Test {
 
     // Kick off a sequence on the sequencer
     await _divSequencer.start(MultiCycleDividerBasicSequence());
+    await _divSequencer.start(MultiCycleDividerEvilSequence(numBits: 32));
```

Comment:
Is it possible to create a narrow divider and do an exhaustive test?

Reply:
Absolutely! That's a good idea - I can add try adding that. Maybe the other (future) PR that I alluded to above?